src/tests/components/IconSearch.test.ts

```
@@ -7,41 +7,106 @@ import { flushPromises } from "../utils/flushPromises";
-
-		const iconButton = screen.getByTestId("icon-button");
-		await fireEvent.click(iconButton);
-
-		expect(mockOnSelectIcon).toHaveBeenCalledWith(mockIcons[0]);
-	});
+    const mockIcons = [
+        { id: "mdi1", label: "MDI Icon 1", origin: "mdi" },
+        { id: "mdi2", label: "MDI Icon 2", origin: "mdi" },
+        { id: "homarr1", label: "Homarr Icon 1", origin: "homarr" },
+        { id: "streamdeck1", label: "StreamDeck Icon 1", origin: "streamdeck" }
+    ] as any[];
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)